src/odemis/acq/stream/_live.py

```diff
@@ -423,9 +438,36 @@ def _compensateShift(self):
         logging.debug("Update metadata for SEM image shift")
         self._detector.updateMetadata({MD_POS_COR: self._shift})
 
-    def _onActive(self, active):
+    @isasync
+    def prepare(self):
+        """
+        Perform calibration if needed
+        """
+        logging.debug("Preparing stream %s ...", self)
```

Comment:
Indicate why it's fine to not call super() or call super(). Reason: because super() only checks for the OPM, which is known to not have any effect for a SEM stream?